playground/nextjs-app-router/components/DemoOptions.tsx

```diff
@@ -49,6 +50,7 @@ export default function DemoOptions({
           {commonOptions}
           <Chain />
           <PaymasterUrl />
+          <IsSponsored />
```

Comment:
can you add this to other relevant components (Swap and Pay)

Reply:
should Pay have `isSponsored` prop ? isn't it always sponsored by Coinbase? or is this a short term thing ?